ci/build_wheel.sh

```diff
@@ -9,9 +9,8 @@ underscore_package_name=$(echo "${package_name}" | tr "-" "_")
 source rapids-configure-sccache
 source rapids-date-string
 
-# Use gha-tools rapids-pip-wheel-version to generate wheel version then
-# update the necessary files
-version_override="$(rapids-pip-wheel-version ${RAPIDS_DATE_STRING})"
+version=$(rapids-generate-version)
```

Comment:
Same question here, will we get RAPIDS or UCX-Py version?

Reply:
See above.